lib/nerves/package.ex

```diff
@@ -213,6 +206,53 @@ defmodule Nerves.Package do
     Path.join(path, @package_config)
   end
 
+  def config(app, path) do
+    project_config =
+      if app == Mix.Project.config[:app] do
+        Mix.Project.config
+      else
+        Mix.Project.in_project(app, path, fn(_mod) ->
+          Mix.Project.config
+        end)
+      end
+    nerves_package =
+      case project_config[:nerves_package] do
+        nil ->
+          # TODO: Deprecated. Clean up after 1.0
+          load_nerves_config(path)
+          config = Application.get_env(app, :nerves_env)
+          Mix.shell.error """
```

Comment:
This shouldn't be a deprecation until we document the new way. My suggestion is to make this silent so that we don't cause every one presenting nerves projects or doing training at ElixirConf to have to update their projects right before the conference. After ElixirConf, I say that we enable the warning so that people know to update their projects.